test/built-ins/RegExp/property-escapes/special-property-value-Script_Extensions-Unknown.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2025 Richard Gibson. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Richard Gibson
+description: >
+  Unicode property "Script_Extensions" and alias "scx" must support
+  special value "Unknown" and alias "Zzzz" (cf.
+  https://www.unicode.org/reports/tr24/#Script_Extensions_Def ).
+esid: sec-compiletocharset
+features: [regexp-unicode-property-escapes]
+---*/
+
+/\p{Script_Extensions=Unknown}/u;
```

Comment:
Does it mean to match anything that isn't in any Script?

Reply:
Yes, see https://www.unicode.org/reports/tr24/#Script_Extensions
> The empty set is not allowed; the scx value for unassigned, private use, and non-character code points is the set { **Unknown** }.

That aspect will be covered once a fix for https://github.com/mathiasbynens/regenerate-unicode-properties/issues/7 is published.